### ISSUE TITLE
POC: using nbd1 as a swap device

### ISSUE THREAD

**Issue by moul** [online-labs/initrd#32]
https://community.cloud.online.net/t/how-to-increase-instance-ram-using-a-swap-device/606

**Comment by consi** [comment]
I think it might be bad idea because when swap will disappear (temporary lost of network connectivity) kernel will panic instantly

**Comment by moul** [comment]
Yes you're right, for now I only want to code the POC to practice some benchmarks but I also think that it's too risky
